dop.sh

```diff
@@ -0,0 +1,18 @@
+## Install helm
```

Comment:
Let us have EKS & DOP & run test cases as a single script
Lets rename this to `verify_dop_on_eks.sh`
GitHub Actions will just trigger this script.